src/Commands/ListAddListCommand.ts

```diff
@@ -0,0 +1,31 @@
+import { NativeCallProperties } from "../Languages/Properties/NativeCallProperties";
+import { NativeCallCommand } from "./NativeCallCommand";
+import { Parameter } from "./Parameters/Parameter";
+import { SingleParameter } from "./Parameters/SingleParameter";
+
+/**
+ * A command that adds everything in one list to another list.
+ */
+export class ListAddListCommand extends NativeCallCommand {
+    /**
+     * Information on parameters this command takes in.
+     */
+    private static parameters: Parameter[] = [
+        new SingleParameter("name", "The name of the list.", true),
+        new SingleParameter("name", "The name of the list.", true)
```

Comment:
These comments aren't really descriptive. They should at least mention something about order, since that's the difference between the lists.